charts/device-plugin-operator/templates/operator.yaml

```diff
@@ -0,0 +1,836 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: inteldeviceplugins-leader-election-role
+  namespace: {{ .Release.Namespace | quote }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
can you update this with the fixes in https://github.com/intel/intel-device-plugins-for-kubernetes/pull/1155

Reply:
done